tools/monorepo-utils/src/code-freeze/commands/changelog/lib/index.ts

```diff
@@ -54,8 +54,8 @@ const updateReleaseChangelogs = async (
 
 	// Convert PR number to markdown link.
 	nextLog = nextLog.replace(
-		/\[#(\d+)\]/g,
-		'[$&](https://github.com/woocommerce/woocommerce/pull/$1)'
+		/\[#(\d+)\](?!\()/g,
```

Comment:
This negative lookahead ensures we don't match `[#111](` so that we don't break existing markdown links in the changelog entry.